<launch>
    <arg name="network_type" default="wifi"/>

    <rosparam command="load" file="$(find turtlebot2i_edge)/config/network.yaml" ns="/"/>
    <param name="/network/type" value="$(arg network_type)"/>

    <node name="network_simulation" pkg="turtlebot2i_edge" type="network_simulation" output="screen">
        <remap from="odom_0" to="/turtlebot2i/odom"/>
        <remap from="odom_1" to="/turtlebot2i_0/odom"/>
        <remap from="odom_2" to="/turtlebot2i_1/odom"/>
        <remap from="odom_3" to="/turtlebot2i_2/odom"/>

        <remap from="stamp_0" to="/turtlebot2i/edge/stamp"/>
        <remap from="stamp_1" to="/turtlebot2i_0/edge/stamp"/>
        <remap from="stamp_2" to="/turtlebot2i_1/edge/stamp"/>
        <remap from="stamp_3" to="/turtlebot2i_2/edge/stamp"/>
    </node>
</launch>